runtime/onert/core/include/ir/train/ITrainableOperation.h

```diff
@@ -39,6 +39,7 @@ class ITrainableOperation : virtual public IOperation
   virtual std::unique_ptr<ITrainableOperation> clone() const = 0;
   virtual void accept(OperationVisitor &v) const override = 0;
   virtual void accept(TrainableOperationVisitor &v) const = 0;
+  virtual bool hasTrainableParameter() const = 0;
```

Comment:
This class is an abstract class so `virtual bool hasTrainablePrameter() const { return false; }` is impossible, right?

Reply:
To provide default implementation, we may remove `= 0`.
```suggestion
  virtual bool hasTrainableParameter() const { return false }
```
It is still abstract class because it has other pure virtual functions.

Reply:
I think making user define it explicitly is better providing default argument in terms of readability